### PR TITLE
refactor: view my nfts loading optimization

### PR DESCRIPTION
**Pull request by lynnshaoyu**
https://uniswaplabs.atlassian.net/browse/WEB-1641?atlOrigin=eyJpIjoiZmNmMWMwMTdjYTNhNDYyNWFhYTMyYzE4OGRmNjI3YWIiLCJwIjoiaiJ9

### CODE REVIEW

**Comment by lynnshaoyu** [comment]
pls hold off on review, making a change... noticed a bug

File: src/nft/pages/profile/profile.tsx
Comment (vm) [inline src/nft/pages/profile/profile.tsx:52]: why this change?

**Review by vm — APPROVED** [review]
lgtm but please have product double check before merge

**Pull request merged**
